content/env-variables.md

```diff
@@ -183,11 +183,11 @@ pub struct ProgramConfig {
 
 The example above shows a hypothetical config account for the NFT staking program example we've referenced throughout the lesson. It stores data representing the token that should be used for rewards and the amount of tokens to give out for each day of staking.
 
-With the config account defined, simply ensure that the rest of your code references this account when using these values. That way, if the data in the account changes, the program adapts accordingly.
+With the config account defined, ensure that the rest of your code references this account when using these values. That way, if the data in the account changes, the program adapts accordingly.
 
-### Constrain config updates to hard-coded admins
+### Constraint config updates to hard-coded admins
 
-You'll need a way to initialize and update the config account data. That means you need to have one or more instructions that only an admin can invoke. The simplest way to do this is to hard-code an admin's public key in your code and then add a simple signer check into your instruction's account validation comparing the signer to this public key.
+You'll need a way to initialize and update the config account data. That means you need to have one or more instructions that only an admin can invoke. The simplest way to do this is to hard-code an admin's public key in your code and then add a simple signer check into your instruction's account validation, comparing the signer to this public key.
```

Comment:
Nah that comma doesn't make sense. Maybe we refactor the entire sentence:
> The simplest way to do this is to hard-code an admin's public key in your code. You can then add a simple signer check into your instruction's account validation that compares the signer to this public key.

Reply:
Yeah,
 it seems to be lengthy as well, I will do as you have suggested.